pkgs/by-name/so/softether/package.nix

```diff
@@ -40,11 +41,11 @@ stdenv.mkDerivation rec {
       Makefile
   '';
 
-  meta = with lib; {
+  meta = {
     description = "Open-Source Free Cross-platform Multi-protocol VPN Program";
     homepage = "https://www.softether.org/";
-    license = licenses.asl20;
-    maintainers = [ maintainers.rick68 ];
+    license = lib.licenses.asl20;
+    maintainers = [ lib.maintainers.rick68 ];
```

Comment:
Please add `aarch64-linux` to `platforms`. Has been verified by myself on Ampere Altra Q64-22.

Reply:
thanks! done